docs/guides/transpiler-stages.ipynb

```diff
@@ -128,7 +128,7 @@
     {
```

Comment:
<p>Ugh, just noticed this chart doesn't really make sense with the following sentence:</p><blockquote>This wide distribution demonstrates how difficult it is for the SWAP mapper to compute the best mapping.</blockquote><p>I'll try with a different seed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
The problem was that the seed was the same for each transpilation. Fixed by moving the pass manager creation inside the loop.